packages/payload-cloud/src/plugin.ts

```diff
@@ -93,5 +95,59 @@ export const payloadCloudPlugin =
       })
     }
 
-    return config
+    // If the user has tasks configured, we set up cron jobs on init.
+    // We also make sure to only run on one instance using a instance identifier stored in a global.
+    if (config.jobs?.tasks) {
+      config.globals = [
+        ...(config.globals || []),
+        {
+          slug: 'payload-cloud-instance',
+          admin: {
+            hidden: true,
+          },
+          fields: [
+            {
+              name: 'instance',
+              type: 'text',
+              required: true,
+            },
+          ],
+        },
+      ]
+      config.onInit = async (payload) => {
```

Comment:
double-check that this works - it looks to me like you are re-binding the `config.onInit`, so that means on line `118`, there will _always_ be a `config.onInit`. Could this call the `onInit` recursively? Instead, we should probably do something like this:

```js
const newOnInit = async (payload) => {
  if (config.onInit) await config.onInit(payload)
  
  // do the rest
}

config.onInit = newOnInit
```